packages/grafana-ui/src/components/SecretFormField/SecretFormField.tsx

```diff
@@ -40,13 +41,15 @@ export const SecretFormField: FunctionComponent<Props> = ({
   inputWidth = 12,
   onReset,
   isConfigured,
+  tooltip,
   placeholder = 'Password',
   ...inputProps
 }: Props) => {
   const styles = getSecretFormFieldStyles();
   return (
     <FormField
       label={label!}
+      tooltip={tooltip!}
```

Comment:
the `!` means it can not be null... is that required here?

Reply:
@ryantxu I was following label but it's def not req. Will test some things out.

Reply:
@ryantxu so I tested it by replacing one of the if/else that needs to be replaced on Azure Monitor and it worked find. It doesn't seem to make it required.